vignettes/images/function-flow.puml

```diff
@@ -47,7 +47,7 @@ card classify_diabetes_status() as cd {
         action "include_hba1c()" as in_hba1c <<inclusion>>
         action "include_diabetes_diagnosis()" as in_diagnosis <<inclusion>>
         action "include_podiatrist_services()" as in_podiatrist <<inclusion>>
-        action "include_purchases_gld()" as in_gld <<inclusion>>
```

Comment:
hmm, but now the `exclude_purchases_of_weight_loss_drugs()` doesn't match.

Reply:
We could rename that to `exclude_weight_loss_drug_purchases` or is that too crazy?

Reply:
Yea, that would work, since we use the abbreviation for GLD anyway. Good!

Reply:
I have changed `weight_loss_drugs` to `wld` in #75